src/main/java/Duke.java

```diff
@@ -1,10 +1,202 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
```

Comment:
Missing javadocs for class. Most of your methods are private though, so they don't need javadocs 👍